release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -151,6 +151,10 @@ In {product-title} {product-version}, instead of using the built-in roles, you c
 
 If you need the virtual IP (VIP) addresses to run on the control plane nodes in a vSphere installation, you must configure the `ingressVIP` addresses to run exclusively on the control plane nodes. By default, {product-title} allows any node in the worker machine configuration pool to host the `ingressVIP` addresses. Because vSphere environments deploy worker nodes in separate subnets from the control plane nodes, configuring the `ingressVIP` addresses to run exclusively on the control plane nodes prevents issues from arising due to deploying worker nodes in separate subnets. For additional details, see xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#configure-network-components-to-run-on-the-control-plane_installing-vsphere-installer-provisioned-network-customizations[Configuring network components to run on the control plane in vSphere].
 
+[id="ocp-4-13-OCP-on-ARM"]
+==== {product-title} on ARM
```

Comment:
```suggestion
==== {product-title} on 64-bit ARM
```

Reply:
This note is a little confusing to me. Is this calling out `Azure user-provisioned installations` or `now supported on 64-bit ARM systems`? 
I guess I'm asking because that could impact the title off the RN. I'm also not sure, then, if the first two sentences should be combined.

Reply:
Gotcha! Yeah I usually did the title like these so that I can show all the features that are now supported on OCP on ARM.